app/webpacker/components/RegistrationsV2/Register/CompetingStep.jsx

```diff
@@ -171,18 +172,23 @@ export default function CompetingStep({
 
   const actionUpdateRegistration = () => {
     confirm({
-      content: i18n.t('competitions.registration_v2.update.update_confirm'),
+      content: i18n.t(competitionInfo.allow_registration_edits ? 'competitions.registration_v2.update.update_confirm' : 'competitions.registration_v2.update.update_confirm_contact'),
     }).then(() => {
-      dispatch(setMessage('competitions.registration_v2.update.being_updated', 'basic'));
-      updateRegistrationMutation({
-        user_id: registration.user_id,
-        competition_id: competitionInfo.id,
-        competing: {
-          comment: hasCommentChanged ? comment : undefined,
-          event_ids: hasEventsChanged ? selectedEvents : undefined,
-        },
-        guests,
-      });
+      if (competitionInfo.allow_registration_edits) {
+        dispatch(setMessage('competitions.registration_v2.update.being_updated', 'basic'));
+        updateRegistrationMutation({
+          user_id: registration.user_id,
+          competition_id: competitionInfo.id,
+          competing: {
+            comment: hasCommentChanged ? comment : undefined,
+            event_ids: hasEventsChanged ? selectedEvents : undefined,
+          },
+          guests,
+        });
+      } else {
+        const updateMessage = `${hasCommentChanged ? `Comment to ${comment} ` : ''}${hasEventsChanged ? `Events to ${selectedEvents.map((eventId) => events.byId[eventId].name).join(', ')} ` : ''}${hasGuestsChanged ? `Guests to ${guests}` : ''}`;
```

Comment:
The text formatting in the prefill could be a lot cleaner. I'd suggest the following 

> I would like to update my registration details as follows:
> Events: 3x3x3, 3bld
> Comment: Here's an example comment
> Guests: 7


And keep the behaviour of not including unchanged fields, that's great - eg if no change to guests is made:

> I would like to update my registration details as follows:
> Events: 3x3x3, 3bld
> Comment: Here's an example comment